### ISSUE TITLE
Provide wpEditor.print for printing to results box from Javascript

### ISSUE THREAD

**Issue by stuhlmueller** [probmods/webppl-editor#25]
This is needed for the Javascript boxes on dippl. I tried the following workaround, but it doesn't reliably print to the correct results box (and instead adds to the results for a previous webppl box):

``` js
var printResult = function(x){
  var container = wpEditor.makeResultContainer();
  container.appendChild(document.createTextNode(x));
}
```

**Comment by longouyang** [comment]
What if I rewrote `print` to work both as a cps function and a regular one?

**Comment by stuhlmueller** [comment]
That would work.

**Comment by longouyang** [comment]
Fixed in 0d5f892

**Comment by stuhlmueller** [comment]
Thanks!
